tests/cli/commands/test_cases.py

```diff
@@ -119,19 +120,19 @@ def test_case_3():
                           datetime=datetime.utcnow(),
                           properties={})
 
-        label_item.ext.enable(Extensions.LABEL)
-        label_item.ext.label.apply(label_description='ML Labels',
-                                   label_type='vector',
-                                   label_properties=['label'],
-                                   label_classes=[
-                                       LabelClasses.create(
-                                           classes=['one', 'two'],
-                                           name='label')
-                                   ],
-                                   label_tasks=['classification'],
-                                   label_methods=['manual'],
-                                   label_overviews=overviews)
-        label_item.ext.label.add_source(image_item, assets=['ortho'])
+        LabelExtension.add_to(label_item)
+        label_extension = LabelExtension.ext(label_item)
```

Comment:
Just a note that you'll be able to replace this with

```python
label_extension = LabelExtension.ext(label_item, add_if_missing=True)
```

after then next release

Reply:
I'll add a TODO.